scipy/sparse/linalg/matfuncs.py

```diff
@@ -592,6 +592,17 @@ def _expm(A, use_exact_onenorm):
     if len(A.shape) != 2 or A.shape[0] != A.shape[1]:
         raise ValueError('expected a square matrix')
 
+    # Trivial case
+    if A.shape == (1, 1):
+
+        # 'ndarray' constructor does not accept
+        # 'data' argument, so we need to call
+        # 'np.array' instead
+        if type(A).__name__ == 'ndarray':
+            return np.asarray([[np.exp(A[0, 0])]])
```

Comment:
Actually, that's a good question.  I was worried about not preserving `ndarray` subclasses.  However, it seems that subclasses aren't preserved in the old behaviour AFAICT.  Thoughts, @ev-br , @pv, @ewmoore ?